exercises/clock/README.md

```diff
@@ -18,11 +18,24 @@ add an `Eq` instance, and implement the functions:
 `addDelta` adds a duration, expressed in hours and minutes, to a given time,
 represented by an instance of `Clock`.
 
+`fromHourMin` takes an hour and minute, and returns an instance of `Clock` with 
+those hours and minutes.
+
+`toString` takes an instance of `Clock` and returns a string representation 
+of the clock, in 0 padded format like "08:03" or "22:35"
+
 You will find a dummy data declaration and type signatures already in place,
 but it is up to you to define the functions and create a meaningful data type,
 newtype or type synonym.
 
+If you need help getting started with Types, take a look at:
+- [Making Our Own Types and Typeclasses][types]
+
+[types]: http://learnyouahaskell.com/making-our-own-types-and-typeclasses
```

Comment:
I don't think this chapter is ideal for addressing this exercise.

The first half of the chapter only deals with `data` definitions, and about halfway through it addresses `type` aliases. But since this exercise is best solved with a `newtype`, and this chapter doesn't even mention `newtype`, it is perhaps more confusing than clarifying.

Having a hyperlink is good, but preferrably to something that focuses on the particular learning goals of the exercise, rather than types in general.

I would recommend either finding another source or removing this hint.

Reply:
Got it, you're right! I used it when I solved the problem but I had to jump down at least halfway because I knew where what I was looking for was. If I can't find a better resource thats more concise I'll remove it.

Reply:
So I found a good resource https://mmhaskell.com/blog/2017/12/24/haskell-data-types-in-5-steps

I am going to replace it for now so let me know what you think. (or if its just better off removed). My motivation to adding something was driven because I didn't know how to get the value of 'hour' or something out of a clock object.